wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -37,12 +41,14 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import java.io.File
 import java.net.URI
 import javax.inject.Inject
+import com.android.build.gradle.BasePlugin as AndroidBasePlugin
 
 class WirePlugin @Inject constructor(
   private val sourceDirectorySetFactory: SourceDirectorySetFactory
 ) : Plugin<Project> {
   private var kotlin = false
   private var java = false
```

Comment:
@jrodbx could those booleans be represented by an enum, so that it could give better semantics in the “when” control flow as well?